Makefile

```diff
@@ -233,6 +236,8 @@ catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
 
 
+
+#==========================local ci toolling not form template=============================
```

Comment:
this is the only change that i did manually

i also filtered out some local changes we had such as using podman instead of docker to only include the content that was relevnet